landlab/components/landslides/landslide_probability.py

```diff
@@ -0,0 +1,535 @@
+#!/usr/env/python
+
+"""
+landslide_probability.py:
+
+Landlab component that simulates relative wetness and probability of failure.
+
+Relative wetness and factor-of-safety are based on the infinite slope
+stability model driven by topographic and soils inputs and recharge provided
+by user in a "landslide_driver" file. In addition, the component simulates
+the probability of failure for each node based on Monte Carlo simulations
+of the factor-of-safety as the number of simulations with factor-of-safety
+<= 1.0 divided by the number of simulations.
+
+Modified to be more generic in the inputs for greater usability as well
+as accomodate functionality with new release of Landlab version 1.
+
+.. codauthor:: R.Strauch, E.Istanbulluoglu, & S.Nudurupati
+University of Washington
+Created on Thu Aug 20, 2015
+Last edit April 04, 2017
+"""
+
+# %% Import Libraries
+from landlab import Component
+from landlab.utils.decorators import use_file_name_or_kwds
+import numpy as np
+from scipy import interpolate
+from statsmodels.distributions.empirical_distribution import ECDF
+import copy
+
+# %% Instantiate Object
+
+
+class LandslideProbability(Component):
+    """
+    Landlab component designed to calculate a probability of failure at
+    each grid node based on the infinite slope stability model
+    stability index (Factor of Safety).
+
+    The driving force for failure is provided by the user in the form of
+    groundwater recharge; 4 options for providing recharge are included.
+    The model uses topographic and soils characteristics provided as input
+    by the user in the landslide_driver.
+
```

Comment:
I'm not sure what the landslide_driver is. Please expand description (here or earlier in the docstring where landslide driver is mentioned). Based on the code it appears that either parameters can be specified as keyword arguments OR parameters can be given in an input file. 

I suspect this is a holdover from when most landlab components were specified with input files.